spec/jobs/users/subscribe_to_mailchimp_newsletter_job_spec.rb

```diff
@@ -7,13 +7,12 @@
     let(:user) { FactoryBot.create(:user) }
 
     it "subscribes user to mailchimp newsletter" do
-      mailchimp_bot = MailchimpBot.new(user)
+      mailchimp_bot = double
+      allow(MailchimpBot).to receive(:new).and_return(mailchimp_bot)
 
-      allow(mailchimp_bot).to receive(:upsert)
+      expect(mailchimp_bot).to receive(:upsert)
```

Comment:
I suggest adding `allow(mailchimp_bot).to receive(:upsert)`, changing expectation to `have_received`, and moving `expect` lower than `perform_now`. It's easier to read and understand like this, and `rubocop-rspec` recommends this.

Reply:
Out of curiosity, why doesn't it run during the build? Would save some review time.

Reply:
Thanks for raising this question. I checked in with the team and it's like this only for historical reasons. We'll work on enabling it.